firefox-ios/Client/Frontend/Microsurvey/Prompt/MicrosurveyPromptView.swift

```diff
@@ -47,12 +47,22 @@ class MicrosurveyPromptView: UIView, ThemeApplicable, Notifiable {
     private var logoWidthConstraint: NSLayoutConstraint?
     private var logoHeightConstraint: NSLayoutConstraint?
 
+    private var logoSizeScaled: CGFloat {
+        return UIFontMetrics.default.scaledValue(for: UX.logoSize)
+    }
+
     private let windowUUID: WindowUUID
     var notificationCenter: NotificationProtocol
 
     private lazy var logoImage: UIImageView = .build { imageView in
         imageView.image = UIImage(imageLiteralResourceName: ImageIdentifiers.homeHeaderLogoBall)
         imageView.contentMode = .scaleAspectFit
+        imageView.isAccessibilityElement = true
+        imageView.accessibilityLabel = String(
+            format: .Microsurvey.Prompt.LogoImageA11yLabel,
+            AppName.shortName.rawValue
+        )
+        imageView.accessibilityIdentifier = AccessibilityIdentifiers.Microsurvey.Prompt.firefoxLogo
```

Comment:
Not a comment for this line but for the `titleLabel` (line 68): Could you please add `label.accessibilityTraits.insert(.header)` for the prompt title?

Reply:
I can't speak to the usual arguments but tell you why I'd add it here:

1. A voice over user would have to navigate the whole website to get to the prompt - even if they are using the rotor set to headings.
2. The other to views following the prompt have one or more headings and the label in the same position is always a heading in these views.

Reply:
okay, let me follow up and will report back!

Reply:
@thatswinnie to clarify on point 1, I was under the assumption that the `layoutChanged` navigates to the prompt regardless if its a header or not as long as we pass in the UI element as an argument to that notification. Please correct me if I am missing something.

Reply:
@cyndichin you are absolutely right. Thanks for clarifying. I had previously only tested with turning voiceover on after the app was launched which lead to having trouble getting to the prompt. If I have voiceover turned on before app launch the prompt headline is getting selected through the notification.

Reply:
great, thank you for checking!